ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Nagios_Plugins/tasks/additional_plugins/check_docker_overlay2_size.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Nagios plugin to check the size of /var/lib/docker/overlay2
+# Allows passing thresholds as parameters
+
+# Default thresholds (in GB)
+DEFAULT_WARN_THRESHOLD=30
+DEFAULT_CRIT_THRESHOLD=40
+
+# Folder to check
+TARGET_DIR="/var/lib/docker/overlay2"
```

Comment:
I'm not sure we have an such situations, but maybe have the `/` on the end so we can see into symlinks if that's been used to put the file system somewhere else without an explicit mount at that location.
```suggestion
TARGET_DIR="/var/lib/docker/overlay2/"
```

Reply:
I agree!